### PR TITLE
Refactoring name of the language (cryptsl->crysl)

### PR DESCRIPTION
**Pull request by gokcekrky**
This pull request has dependency on Cryptsl pull request [#31](https://github.com/CROSSINGTUD/CryptSL/issues/31) and Crypto-API-Rules pull request [#60](https://github.com/CROSSINGTUD/Crypto-API-Rules/pull/60).

Changes:
* All accurences of cryptsl is refactored. (cryptsl->crysl).
* Versions numbers are updated.

### CODE REVIEW

File: CryptoAnalysis-Android/pom.xml [hunk 1]
```diff
@@ -4,11 +4,11 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>de.fraunhofer.iem</groupId>
 	<artifactId>CryptoAnalysis-Android</artifactId>
-	<version>2.4-SNAPSHOT</version>
+	<version>3.0-SNAPSHOT</version>
```
Comment (AnakinSklavenwalker) [inline CryptoAnalysis-Android/pom.xml]: **TBD** i'm not sure yet if we want to upgrade to 3.0 or just update to 2.7.

**Comment by AnakinSklavenwalker** [comment]
i'll review the changes and corss checks if everything works on my machine.
@kruegers As far as i can remember we decided to merge this NOT in this year, right?

File: CryptoAnalysis-Android/pom.xml [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<cryptoAnalysisVersion>2.4-SNAPSHOT</cryptoAnalysisVersion>
+		<cryptoAnalysisVersion>3.0-SNAPSHOT</cryptoAnalysisVersion>
```
Comment (AnakinSklavenwalker) [inline CryptoAnalysis-Android/pom.xml]: TBD i'm not sure yet if we want to upgrade to 3.0 or just update to 2.7.

File: CryptoAnalysis/pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>de.fraunhofer.iem</groupId>
 	<artifactId>CryptoAnalysis</artifactId>
-	<version>2.6-SNAPSHOT</version>
+	<version>3.0-SNAPSHOT</version>
```
Comment (AnakinSklavenwalker) [inline CryptoAnalysis/pom.xml]: TBD i'm not sure yet if we want to upgrade to 3.0 or just update to 2.7.

File: CryptoAnalysis-Android/src/main/java/de/fraunhofer/iem/crypto/CogniCryptAndroidAnalysis.java [hunk 4]
```diff
@@ -143,16 +143,16 @@ private void prepareAnalysis() {
...
     }
 
-	protected List<CryptSLRule> getRules() {
-		List<CryptSLRule> rules = Lists.newArrayList();
+	protected List<CrySLRule> getRules() {
+		List<CrySLRule> rules = Lists.newArrayList();
 		if (rulesDirectory == null) {
 			throw new RuntimeException(
-					"Please specify a directory the CrySL rules (.cryptslbin Files) are located in.");
+					"Please specify a directory the CrySL rules (.cryslbin Files) are located in.");
```
Comment (AnakinSklavenwalker) [inline CryptoAnalysis-Android/src/main/java/de/fraunhofer/iem/crypto/CogniCryptAndroidAnalysis.java]: Please use static constant from CrySLModelReader.java

File: CryptoAnalysis-Android/src/main/java/de/fraunhofer/iem/crypto/CogniCryptAndroidAnalysis.java [hunk 5]
```diff
@@ -143,16 +143,16 @@ private void prepareAnalysis() {
...
 		}
 		File[] listFiles = new File(rulesDirectory).listFiles();
 		for (File file : listFiles) {
-			if (file != null && file.getName().endsWith("cryptslbin")) {
-				rules.add(CryptSLRuleReader.readFromFile(file));
+			if (file != null && file.getName().endsWith("cryslbin")) {
```
Comment (AnakinSklavenwalker) [inline CryptoAnalysis-Android/src/main/java/de/fraunhofer/iem/crypto/CogniCryptAndroidAnalysis.java]: Definitely use static constant from CrySLModelReader.java here. 
*Note: It's OK to use the dot '.' character with endsWith(string)*

File: CryptoAnalysis/src/main/java/crypto/analysis/CrySLRulesetSelector.java [hunk 6]
```diff
@@ -6,16 +6,16 @@
...
+import crypto.rules.CrySLRuleReader;
+import crypto.cryslhandler.CryslReaderUtils;
 import org.apache.commons.io.FilenameUtils;
 
 public class CrySLRulesetSelector {
 	public static enum RuleFormat {
 		SOURCE() {
 			public String toString() {
-				return".cryptsl";
+				return".crysl";
```
Comment (AnakinSklavenwalker) [inline CryptoAnalysis/src/main/java/crypto/analysis/CrySLRulesetSelector.java]: Please use static constant from CrySLModelReader.java

**Comment by gokcekrky** [comment]
I made required changes

**Pull request merged**